lighthouse-core/test/audits/viewport-test.js

```diff
@@ -38,4 +38,16 @@ describe('Mobile-friendly: viewport audit', () => {
       Viewport: 'width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1'
     }).rawValue, true);
   });
+
+  it('passes when a viewport with spaces is provided', () => {
```

Comment:
I’m happy either way, whichever would better fit your existing tests; I assume a failure would be as verbose either way. Pushed change.